docs/narr/project.rst

```diff
@@ -856,12 +863,16 @@ also informs Python that the directory which contains it is a *package*.
    WSGI application.  This function is meant to be called by the
    :term:`PasteDeploy` framework as a result of running ``pserve``.
 
-   Within this function, application configuration is performed.
+   Before ``main`` is called :term:`PasteDeploy` :ref:`configures logging <myproject_ini_logging>`.
+   Within ``main`` further application configuration is performed.
 
    Line 7 opens a context manager with an instance of a :term:`Configurator`.
+   This applies lines in the ``.ini`` file to ``config``, the configuration returned by :term:`Configurator`.
+   Only those lines in :ref:`[app:main] <app_main>` which are understood by :app:`Pyramid` are applied, such as ``pyramid.includes = pyramid_debugtoolbar``.
 
-   Line 8 adds support for Jinja2 templating bindings, allowing us to
-   specify renderers with the ``.jinja2`` extension.
+   Line 8 adds support for Jinja2 templating bindings, allowing us to specify renderers with the ``.jinja2`` extension.
+   The expectation is that the add-on applies what is relevant in the ``.ini`` file at this point.
+   Consult your add-on's documentation.
```

Comment:
settings come from the settings field passed to the config via `Configurator(settings=settings)`. You choose to pass the ones from the ini, but I'm not comfortable just saying everywhere that things are from the ini file.